{"uid":"ca9f77ac7321cda7","name":"Validate Council Output -- @1.10 Testing : ChelmsfordCityCouncil","fullName":"Test each council output matches expected results in /outputs: Validate Council Output","historyId":"62e27e72c1bfb789c6c8d04a24090df5","time":{"start":1676969790590,"stop":1676969790651,"duration":61},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='mychelmsford.secure.force.com', port=443): Max retries exceeded with url: /WasteServices/WM_WasteViewProperty?id=XXXXXXXXXXXXXXX&User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F108.0.0.0+Safari%2F537.36 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f682f855310>: Failed to establish a new connection: [Errno -2] Name or service not known'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 34, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 67, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 56, in template_method\n    page = self.get_data(address_url)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 84, in get_data\n    full_page = requests.get(url, headers, verify=False)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='mychelmsford.secure.force.com', port=443): Max retries exceeded with url: /WasteServices/WM_WasteViewProperty?id=XXXXXXXXXXXXXXX&User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F108.0.0.0+Safari%2F537.36 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f682f855310>: Failed to establish a new connection: [Errno -2] Name or service not known'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 34, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 67, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 56, in template_method\n    page = self.get_data(address_url)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 84, in get_data\n    full_page = requests.get(url, headers, verify=False)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","steps":[{"name":"Given the council: \"ChelmsfordCityCouncil\"","time":{"start":1676969790590,"stop":1676969790591,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When we scrape the data from \"ChelmsfordCityCouncil\"","time":{"start":1676969790591,"stop":1676969790650,"duration":59},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='mychelmsford.secure.force.com', port=443): Max retries exceeded with url: /WasteServices/WM_WasteViewProperty?id=XXXXXXXXXXXXXXX&User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F108.0.0.0+Safari%2F537.36 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f682f855310>: Failed to establish a new connection: [Errno -2] Name or service not known'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 34, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 67, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 56, in template_method\n    page = self.get_data(address_url)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 84, in get_data\n    full_page = requests.get(url, headers, verify=False)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},{"name":"Then the result is valid json","time":{"start":1676969790651,"stop":1676969790651,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"And the output should validate against the schema","time":{"start":1676969790651,"stop":1676969790651,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Test each council output matches expected results in /outputs"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"council","value":"ChelmsfordCityCouncil"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"ba777d1c376ff629","reportUrl":"https://robbrad.github.io/UKBinCollectionData/3.9/487//#testresult/ba777d1c376ff629","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='mychelmsford.secure.force.com', port=443): Max retries exceeded with url: /WasteServices/WM_WasteViewProperty?id=XXXXXXXXXXXXXXX&User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F108.0.0.0+Safari%2F537.36 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f98b5f57340>: Failed to establish a new connection: [Errno -2] Name or service not known'))\n","time":{"start":1676969662928,"stop":1676969663015,"duration":87}},{"uid":"842225ac76841e69","reportUrl":"https://robbrad.github.io/UKBinCollectionData/3.9/484//#testresult/842225ac76841e69","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='mychelmsford.secure.force.com', port=443): Max retries exceeded with url: /WasteServices/WM_WasteViewProperty?id=XXXXXXXXXXXXXXX&User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F108.0.0.0+Safari%2F537.36 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4224671310>: Failed to establish a new connection: [Errno -2] Name or service not known'))\n","time":{"start":1676966956131,"stop":1676966956194,"duration":63}}]},"tags":[]},"source":"ca9f77ac7321cda7.json","parameterValues":["ChelmsfordCityCouncil"]}